tmcprototype/CspSubarrayLeafNode/CspSubarrayLeafNode.xmi

```diff
@@ -31,7 +31,6 @@
     <deviceProperties name="CspSubarrayNodeFQDN" description="FQDN of the CSP Subarray Node Tango Device Server.">
       <type xsi:type="pogoDsl:StringType"/>
       <status abstract="false" inherited="false" concrete="true" concreteHere="true"/>
-      <DefaultPropValue>mid_csp/elt/subarray01</DefaultPropValue>
     </deviceProperties>
```

Comment:
Is it verified that the xmi file loads in POGO properly after removing the tag <DefaultPropValue> entirely? The tag could be mandatory with blank value.

Reply:
We have verified. Code is generating properly in POGO. There is no need of tag with blank value.

Reply:
Commit ID: 9ab1a8bbeec93788ee716d8c14cabd8cb9e7a1bd